features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -252,6 +252,24 @@
   step %{I should see "Work was successfully posted"}
 end
 
+When /^an assignment has been fulfilled in a gift exchange$/ do
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.